src/NGen.cpp

```diff
@@ -146,6 +147,21 @@ int main(int argc, char *argv[]) {
     double t = 0;
     time_step_t dt = 3600; //tshirt time step
 
+    // TODO: parameterize these values, rather than hard-code
+    // TODO: current mapping file shows "wat-*" rather than "cat-*" for some reason ... figure out if this has further implications
```

Comment:
waterbodies and catchments are independent entities that are related.  comids associate with waterbodies, catchments realize waterbodies.

Reply:
I made some inferences about the nature of the objects represented by the JSON data and how relationships between things work.  Given the mapping was between those identifiers (i.e., `wat-*`) and the identifiers/keys of the JSON objects holding GIUH data,  I expected all these things were particular to catchment representations. 

If something like `wat-87` is actually a waterbody id, that make sense, but is it not the case that, e.g., object _"9731248"_ in `GIUH.json` describes data pertaining to a catchment (as opposed to a waterbody)?  If that is indeed the case, where is the relationship to a catchment id (in the form that NGen seems to start with; i.e., `cat-*`) defined?